rust/cbindgen.toml

```diff
@@ -82,6 +82,7 @@ include = [
     "QuicState",
     "QuicTransaction",
     "FtpEvent",
+    "FtpCommand",
```

Comment:
Is this needed? Doesn't appear to actually get included in the generated header.